waspc/cli/Command/Telemetry.hs

```diff
@@ -24,7 +24,21 @@ telemetry = do
     telemetryDisabled <- liftIO isTelemetryDisabled
     waspSaysC $ "Telemetry is currently: " <> (if telemetryDisabled
         then "DISABLED"
-        else "ENABLED") 
+        else "ENABLED")
+    
+    -- Code's being duplicated again, need some suggestions to improve this.
+    telemetryCacheDirPath <- liftIO ensureTelemetryCacheDirExists
+    maybeProjectHash <- (Just <$> TlmProject.getWaspProjectPathHash) `catchError` const (return Nothing)
+    let maybeProjectHash = case maybeProjectHash of
+                Nothing          -> Nothing                       -- probably should be reporting an error here 
```

Comment:
The question is, what do we want to do?
getWaspProjectHash will throw error if we are not in wasp project.
What do we want to do if we are not in wasp project? Data is sent per-project, so we don't really have anything to report. I think in that case it is fine to return Nothing here, and as a consequence, not print anything about the time of last telemetry data.